crates/ruff_linter/resources/test/fixtures/ruff/RUF052.py

```diff
@@ -37,6 +37,17 @@ def fun():
     _x = "reassigned global"
     return _x
 
+# (we had a false positive when a global var is used like this in 2 functions)
```

Comment:
(am I overcommenting? I was just worried that it might not be clear that this is a single "test" spread across 2 functions, but happy to remove)